finance/bitcoin/Portfile

```diff
@@ -62,6 +62,9 @@ configure.args-append   --with-daemon=no
 configure.args-append   --with-gui=no
 configure.args-append   --enable-wallet=no
 
+# https://trac.macports.org/ticket/55639#comment:3
```

Comment:
Rather than referencing the ticket, I'd suggest just two lines of comment explaining the purpose of the following line. Do you think you could change it to do that and squish the commits?

Reply:
(The rationale being that it will be much easier for the reader not to have to cut and paste the link and then try to parse the whole discussion chain, but rather to understand the issue right here while reading.)